resources/definitions/koonovo_elf.def.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": "KOONOVO ELF",
+    "version": 2,
+    "inherits": "koonovo_base",
+    "overrides": {
+        "machine_name": { "default_value": "KOONOVO ELF" },
+        "machine_width": { "default_value": 310 },
+        "machine_depth": { "default_value": 310 },
+        "machine_height": { "default_value": 345 },
```

Comment:
Alibaba lists the specs of this printer as being 300x300x330: https://www.alibaba.com/product-detail/3d-Desktop-High-precision-3D-Printer_62404382636.html?spm=a2700.galleryofferlist.normal_offer.d_title.bae94c94i7W5gy